test/root/predicates/ERC20Predicate.test.js

```diff
@@ -88,7 +118,7 @@ contract('ERC20Predicate', async function(accounts) {
         contracts.ERC20Predicate.startExitForOutgoingErc20Transfer, headerNumber, blockProof, block.number, block.timestamp, reference, 1, /* logIndex */ exitTx)
       const logs = logDecoder.decodeLogs(startExitTx.receipt.rawLogs)
       // console.log(startExitTx, logs)
-      const log = logs[1]
+      const log = logs[utils.filterEvent(logs, 'ExitStarted')]
```

Comment:
lol this next line is stupid 😉 `logs[utils.filterEvent(logs, 'ExitStarted')]==log.event.should.equal('ExitStarted')`.